test/commands/members/index.js

```diff
@@ -34,7 +34,7 @@ b@heroku.com  collaborator
     .then(() => api.done());
   });
 
-  it('shows member when role is viewer', () => {
+  it('does not show member when role is viewer', () => {
```

Comment:
@eablack I explained in the description of this pr that this was just to reflect the change and this could be removed entirely. I'll do that :stuck_out_tongue_winking_eye: